### ISSUE TITLE
DataResolver is a little noisy on application startup

### ISSUE THREAD

**Issue by agrobbin** [ViewComponent/lookbook#458]
### Describe the bug

I'm not sure when this started happening, but I recently noticed some output on application startup (across seemingly all environments) from Lookbook. For each parameter with Ruby code in it:

```ruby
# @param size [Symbol] select {{ UI::CardComponent::SIZES.keys }}
```

We're seeing this output in our application startup logs, coming from [`DataResolver#evaluate`](https://github.com/ViewComponent/lookbook/blob/bd1d3be6d575b9caeca735777f43e098b84db9e2/lib/lookbook/services/data/resolvers/data_resolver.rb#L43C9-L56):

```
Data cannot be evaluated (Input: 'UI::CardComponent::SIZES.keys ')
```

### To Reproduce

Write a component preview with a parameter that has evaluated Ruby, then start the server!

### Expected behavior

There should be no output on application startup.

### Version numbers 

Please complete the following information:

* Lookbook: 2.0.2
* ViewComponent: 3.0.0
* Rails: 7.0.4.3
* Ruby: 3.2.2

**Comment by allmarkedup** [comment]
Okay this should be fixed now in #468 - will go out in the next release if everything looks okay :-)

**Comment by agrobbin** [comment]
Great, thanks for the quick resolution @allmarkedup!

**Comment by allmarkedup** [comment]
I've just released the fix for this in 2.0.3 so I'm going to go ahead and close this down now. Thanks for bringing it to my attention @agrobbin :-)

**Comment by agrobbin** [comment]
Much quieter, thanks @allmarkedup!
